deepchem/models/torch_models/gnn.py

```diff
@@ -400,6 +421,23 @@ def _prepare_batch(self, batch):
 
         return inputs, labels, weights
 
+    def default_generator(
+            self,
```

Comment:
Can you add a docstring here? It should explain why we need a custom default generator here instead of the default